ci/h2spec.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+set -eu
+set -o pipefail
+shopt -s extglob
+
+SEP="$(tput bold)⣷$(tput sgr0)"
+LOGFILE="l5d-h2spec.log"
+SPIN='⣾⣽⣻⢿⡿⣟⣯⣷'
+
+spin() {
+    local pid=$!
+    local i=0
+    while [ "$(ps a | awk '{print $1}' | grep $pid)" ]; do
+        i=$(( (i+1) %8 ))
+        printf "\r$(tput bold)${SPIN:$i:1}$(tput sgr0)"
+        sleep .1
+    done
+}
+
+echo "${SEP} running h2spec against linkerd..."
+# if we can't find h2spec, download it
+if ! [ -e "h2spec" ] ; then
+    # if we don't already have a h2spec executable, wget it from github
+    echo "${SEP} h2spec not found..."
+    # detect OS for downloading h2spec
+    case $OSTYPE in
+        darwin*)
+          echo "${SEP} detected OS as macOS"
+          H2SPEC_FILE="h2spec_darwin_amd64.tar.gz"
+          ;;
+        linux*)
+          echo "${SEP} detected OS as linux"
+          H2SPEC_FILE="h2spec_linux_amd64.tar.ghz"
+          ;;
+
+    esac
+    H2SPEC_URL="https://github.com/summerwind/h2spec/releases/download/v2.1.0/${H2SPEC_FILE}"
+    printf "${SEP} downloading h2spec..."
+    wget "${H2SPEC_URL}" > /dev/null 2>&1 &
```

Comment:
on my system i needed to to `wget -O $H2SPEC_FILE...` otherwise i end up with crazy long filenames with `Amz-Credential`'s in it.